src/System.Collections.Specialized/tests/HybridDictionary/CtorTests.cs

```diff
@@ -3,206 +3,66 @@
-            hd.Add(lbls[1], bs[1]);
-            if (hd.Count != 2)
-            {
-                Assert.False(true, string.Format("Error, Count returned {0} instead of 2", hd.Count));
-            }
-            if (!hd.Contains(lbls[0]))
-            {
-                Assert.False(true, string.Format("Error, doesn't contain 1st special item"));
-            }
-            if (!hd.Contains(lbls[1]))
-            {
-                Assert.False(true, string.Format("Error, doesn't contain 2nd special item"));
-            }
-            if (hd.Values.Count != 2)
-            {
-                Assert.False(true, string.Format("Error, Values.Count returned {0} instead of 2", hd.Values.Count));
-            }
+        public void Ctor_Int_Bool(int initialSize, bool caseInsensitive)
+        {
+            HybridDictionary hybridDictionary = new HybridDictionary(initialSize, caseInsensitive);
+            VerifyCtor(hybridDictionary, caseInsensitive: caseInsensitive);
+        }
 
-            hd.Remove(lbls[1]);
-            if (hd.Count != 1)
-            {
-                Assert.False(true, string.Format("Error, failed to remove item"));
-            }
-            if (hd.Contains(lbls[1]))
-            {
-                Assert.False(true, string.Format("Error, failed to remove special item"));
-            }
+        public static void VerifyCtor(HybridDictionary hybridDictionary, bool caseInsensitive)
+        {
+            Assert.Equal(0, hybridDictionary.Count);
+            Assert.Equal(0, hybridDictionary.Keys.Count);
+            Assert.Equal(0, hybridDictionary.Values.Count);
 
-            //
-            // [] many elements not overriding Equals()
-            //
-            hd.Clear();
-            lbls = new Hashtable[BIG_LENGTH];
-            bs = new ArrayList[BIG_LENGTH];
-            for (int i = 0; i < BIG_LENGTH; i++)
-            {
-                lbls[i] = new Hashtable();
-                bs[i] = new ArrayList();
-                hd.Add(lbls[i], bs[i]);
-            }
-            if (hd.Count != BIG_LENGTH)
-            {
-                Assert.False(true, string.Format("Error, Count returned {0} instead of {1}", hd.Count, BIG_LENGTH));
-            }
-            for (int i = 0; i < BIG_LENGTH; i++)
-            {
-                if (!hd.Contains(lbls[0]))
-                {
-                    Assert.False(true, string.Format("Error, doesn't contain 1st special item"));
-                }
-            }
-            if (hd.Values.Count != BIG_LENGTH)
-            {
-                Assert.False(true, string.Format("Error, Values.Count returned {0} instead of {1}", hd.Values.Count, BIG_LENGTH));
-            }
+            Assert.False(hybridDictionary.IsFixedSize);
+            Assert.False(hybridDictionary.IsReadOnly);
+            Assert.False(hybridDictionary.IsSynchronized);
 
-            hd.Remove(lbls[0]);
-            if (hd.Count != BIG_LENGTH - 1)
+            if (caseInsensitive)
```

Comment:
Here's case sensitivity tests for the getter but I didn't see them for Contains.